code/src/index.css

```diff
@@ -1,13 +1,156 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  display: flex;
+  justify-content: center;
+  margin-top: 50px;
+  font-family: 'Cutive Mono', monospace;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+
+/*The form*/
+
+.happy-form {
+border: solid gray 2px;
+width: 400px;
+padding-top: 20px;
+padding-bottom: 10px;
+padding-left: 20px;
+padding-right: 20px ;
+background-color: #f1efef;
+box-shadow: 8px 8px black;
+margin-bottom: 70px;
+}
+
+.happy-form h3 {
+  font-size: 20px;
+}
+
+.form {
+display: flex;
+flex-direction: row-reverse;
+justify-content:space-between;
+align-items: center;
+margin-top:20px;
+margin-bottom:0px;
+width: 100%;
+}
+
+.form-text  {
+  width: 100%;
+  height: 90px;
+  font-size: 15px;
+}
+
+.form-footer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 10px;
+}
+
+.form-button {
+  margin-top: 10px;
+  border-radius: 20px;
+  border: none;
+  background-color: #faafae;
+  padding: 5px 15px 5px 15px;
+  font-size: 15px;
+}
+
+.thought-length {
+  font-size: 11px;
+  color: grey;
+}
+
+/*The messages*/
+
+.like {
+display: flex;
+justify-content: center;
+align-items: center;
+width: 40px;
+height: 40px;
+background-color: rgba(221, 217, 217, 0.667);
+border-radius: 50%;;
+border:none;
+}
+
+.total-likes {
+color: grey;
+margin-left: 5px;
+
+}
+
+.likes {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  
+}
+
+.message-time {
+font-size: 15px;
+color: grey;
+margin-left: 10px;
+}
+
+
+.happy-thought {
+border:solid grey 1px;
+width: 400px;
+padding-top: 20px;
+padding-bottom: 5px;
+padding-left: 10px;
+padding-right: 20px ;
+background-color: #f1efef;
+box-shadow: 8px 8px black;
+margin-bottom: 30px;
+display: flex;
+flex-direction: column;
+}
+
+.message {
+  font-size: 20px;
+  margin-top: 20px;
+  padding-left: 10px;
+}
+
+
+.likes button {
+border:none;
+}
+
+.thought-footer {
+  display: flex;
+  flex-direction: row ;
+  justify-content: space-between;
+  align-items: center;
+  margin-top:20px;
+}
+
+button[type="submit"]:disabled {
+  color: grey;
+}
+
+button[type="submit"]:enabled {
+  color: red;
+}
+
+
+.like[type="submit"]:focus {
+  background-color: #ffadad;
+}
+
+
+.rubrik {
+padding-bottom: 20px;
+font-size: 15px;
+}
```

Comment:
Here are also some indentation and empty lines what need fix and here is a hint for you: if there a happy thought what is not fitting the the box use this line of code: word-wrap: break-word (: